sklearn/_distributor_init.py

```diff
@@ -0,0 +1,10 @@
+""" Distributor init file
+
+Distributors: you can add custom code here to support particular distributions
+of scikit-learn.
+
+For example, this is a good place to put any checks for hardware requirements.
+
+The numpy standard source distribution will not put code in this file, so you
```

Comment:
```suggestion
The scikit-learn standard source distribution will not put code in this file, so you
```

Reply:
Thanks @jeremiedbb, I committed the change